src/js/view/components/LogViewerList.js

```diff
@@ -35,7 +35,7 @@ const LogViewerList = props => {
   const oneCharacterSizeRef = useRef();
   const [characterDimensions, setCharacterDimensions] = useState({
     width: 10,
-    height: 17
+    height: 19
```

Comment:
Has the height of a character changed? Might be beacuse we switched fonts when moving to Fabric UI I guess?

Reply:
I had it changed from 19 to 17 in an earlier commit since I thought it made a difference then. I tried changing it back to solve things now but did not notice any difference. Looking at the program on the laptop and on the bigger screen has caused some issues with the line numbers for me from time to time. Thought I'd leave it in the state it was before I started messing around with it for now.

Reply:
I did not try the trackpad. Should have thought of that. Good thing you noticed.